packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.16.3
+
+* Adds onWebViewRenderProcessTerminated WebViewClient callback on Android to support
```

Comment:
Changelog entries in an Android-specific package don't need to say "on Android".